docs/en/token_endpoint.rst

```diff
@@ -225,6 +225,15 @@ ID Token
 ++++++++
 
 The ID Token is a JSON Web Token (JWT) that contains information on the user that has executed the authentication. The RPs MUST validate the ID Token.
+
```

Comment:
Only for SPID: if some user attributes are requested with scope or claim parameter, the RP metadata SHOULD contains id_token_encrypted_response_alg and id_token_encrypted_response_enc and their value SHOULD NOT be 'none'.
If some user attributes are requested with scope or claim and the RP metadata does not contains id_token_encrypted_response_alg and id_token_encrypted_response_enc or their value is 'none', none user attributes are returned in the id_token from token endpoint response.

Reply:
here my proposal

https://github.com/italia/spid-cie-oidc-docs/issues/82

so if you agree we may merge it as it is and then create a separate PR for that